README.md

```diff
@@ -1,66 +1,228 @@
-# terraform-provider-oneview
+# HPE OneView SDK for Terraform Provider
 
-[![Build Status](https://travis-ci.org/HewlettPackard/terraform-provider-oneview.svg?branch=master)](https://travis-ci.org/HewlettPackard/terraform-provider-oneview)
+## Build Status 
 
-A Terraform provider for oneview
+| 5.60 Branch   | 5.50 Branch   | 5.40 Branch   | 5.30 Branch   |
```

Comment:
We need to add OV and SDK version, please refer python or Chef SDK's

Reply:
This is a first release of terraform 12 so no prior release tags available

Reply:
Done